tcf_core/context_processors.py

```diff
@@ -12,7 +12,7 @@ def base(request):
     return {
         "DEBUG": settings.DEBUG,
         "USER": request.user,
-        "LATEST_SEMESTER": Semester.latest(),
+        "LATEST_SEMESTER": Semester().latest(),
```

Comment:
This change was originally made since Semester.latest() was a default value for some parameters for the functions I wrote. I changed that logic and forgot to revert the change, but its back now.